src/ui/server/controllers/insurance/your-buyer/validation/rules/address.test.ts

```diff
@@ -0,0 +1,48 @@
+import generateValidationErrors from '../../../../../helpers/validation';
+import { FIELDS } from '../../../../../content-strings/fields/insurance/your-buyer';
+import { ERROR_MESSAGES } from '../../../../../content-strings';
+import { addressRules } from './address';
+
+const {
+  INSURANCE: {
+    YOUR_BUYER: { BUYER_ADDRESS },
+  },
+} = ERROR_MESSAGES;
+
+const { YOUR_BUYER } = FIELDS;
+
+describe('controllers/insurance/your-buyer/validation/address/', () => {
+  const mockErrors = {
+    summary: [],
+    errorList: {},
+  };
+
+  describe('when address name field  is not provided', () => {
+    it('should return validation error', () => {
+      const mockSubmittedData = { organisation: 'xdfd', country: 'XAD', address: '' };
```

Comment:
Can we update and use `fixtures/application` for these values please so that we have the values we submit with in tests all in one place?